src/commands/ps.rs

```diff
@@ -0,0 +1,76 @@
+use crate::{cgroups, container::Container, utils};
+use anyhow::{bail, Result};
+use clap::{self, Clap};
+use std::{path::PathBuf, process::Command};
+#[derive(Clap, Debug)]
+pub struct Ps {
+    #[clap(short, long, default_value = "table")]
+    format: String,
+    pub container_id: String,
+    #[clap(setting = clap::ArgSettings::Last)]
+    ps_options: Vec<String>,
+}
+impl Ps {
+    pub fn exec(&self, root_path: PathBuf, systemd_cgroup: bool) -> Result<()> {
+        let container_root = root_path.join(&self.container_id);
+        if !container_root.exists() {
+            bail!("{} doesn't exist.", self.container_id)
+        }
+        let container = Container::load(container_root)?.refresh_status()?;
+        if container.root.exists() {
+            let config_absolute_path = container.root.join("config.json");
+            log::debug!("load spec from {:?}", config_absolute_path);
+            let spec = oci_spec::Spec::load(config_absolute_path)?;
+            log::debug!("spec: {:?}", spec);
+            let cgroups_path = utils::get_cgroup_path(&spec.linux.cgroups_path, container.id());
+            let cmanager = cgroups::common::create_cgroup_manager(cgroups_path, systemd_cgroup)?;
```

Comment:
If the container was created with systemd as cgroup manager, then we need to use systemd in ps as well. You can use the container state get this information. 

```suggestion
            let use_systemd = container.systemd().context("could not determine cgroup manager")?;
            let cmanager = cgroups::common::create_cgroup_manager(cgroups_path, use_systemd)?;
```

Reply:
thanks,changed as suggested.